homeassistant/components/cover/__init__.py

```diff
@@ -236,6 +247,11 @@ def is_closing(self):
     def is_closed(self):
         """Return if the cover is closed or not."""
         raise NotImplementedError()
+        
+    @property
+    def is_tilt_closed(self):
+        """Return if the cover is tilted closed or not."""
+        raise NotImplementedError()
```

Comment:
We already define that 0 tilt means tilt closed, right? So why should this method be re-implemented in every integration when the behavior is clearly defined?

Reply:
The MQTT platform allows the "closed" or minimum tilt to be specified in the config, so it's not necessarily 0.   We could define have a default implementation that defines closed as == 0 but allow it to be overridden if desired

Reply:
Agree with @OttoWinter.  `0` means closed for the position related attributes.  If MQTT does something different, it should be handled in that platform and not employ a design change that requires all the other platforms to implement.

Reply:
Ok, will add a default implementation and remove from platforms that match that

Reply:
I think what we're saying is that the property isn't necessary and should be removed all together.  Please see the comment about opening an arch issue to discuss and get alignment.